### PR TITLE
feat: add confirmation in passcode to password login

### LINKED ISSUE #1057: Add a confirmation dialog in forgot pin.
**Summary:**

When user clicks on forgot pin, he is directed to login page.

**Expected behavior:**

He should get a confirmation dialog.
https://github.com/openMF/mobile-wallet/issues/327

### PR DESCRIPTION
**Pull request by shiv07tiwari**
Fixes #1057

Please Add Screenshots If there are any UI changes.

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by shiv07tiwari** [comment]
videotogif_2019 02 21_02 34 57

**Comment by shiv07tiwari** [comment]
@miPlodder @therajanmaurya Kindly review this.

File: app/src/main/java/org/mifos/mobilebanking/ui/activities/PassCodeActivity.java [hunk 1]
```diff
@@ -51,10 +53,25 @@ public void startNextActivity() {
 
     @Override
     public void startLoginActivity() {
-        Intent i = new Intent(PassCodeActivity.this, LoginActivity.class);
-        i.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TASK);
-        startActivity(i);
-        finish();
+        AlertDialog.Builder builder = new AlertDialog.Builder(PassCodeActivity.this);
```
Comment (therajanmaurya) [inline app/src/main/java/org/mifos/mobilebanking/ui/activities/PassCodeActivity.java]: Use Material Dialog class in utils/ directory

**Review by therajanmaurya — CHANGES_REQUESTED** [review]
Fix the conflicts and make changes as requested

**Comment by shiv07tiwari** [comment]
@therajanmaurya Made the changes.

**Comment by shiv07tiwari** [comment]
@therajanmaurya Fixed the build

**Comment by shiv07tiwari** [comment]
@therajanmaurya Fixed the merge conflict.

**Comment by luckyman20** [comment]
Closing this as the issue is already solved.

**Pull request closed without merging**